cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -19,21 +19,11 @@
 # managed result is of that. Start everything below that directory.
 KUBECTL=${KUBECTL_BIN:-/usr/local/bin/kubectl}
 
-ADDON_CHECK_INTERVAL_SEC=${TEST_ADDON_CHECK_INTERVAL_SEC:-600}
+ADDON_CHECK_INTERVAL_SEC=${TEST_ADDON_CHECK_INTERVAL_SEC:-10}
```

Comment:
Unconstrained, the addon manager consumes a full cpu for a few seconds every iteration. This is a documentation shortcoming:

https://github.com/mikedanese/kubernetes/blob/addon-manager/cluster/addons/addon-manager/kube-addon-update.sh#L36

To mitigate this, we set a 10 minute interval between runs (a value that needs to be adjusted in e2e because it is so unresponsive). Now that we are running the kube addon manager in a container we can deprioritize it by giving it a very small cpu share while making it more responsive.